packages/facility-server/app/routes/apiv1/patient/patientProgramRegistration/patientProgramRegistrationConditions.js

```diff
@@ -84,27 +56,3 @@ patientProgramRegistrationConditions.get(
     });
   }),
 );
-
-patientProgramRegistrationConditions.delete(
```

Comment:
Deleting a patientProgramRegistrationConditions directly is no longer supported

Reply:
Yes that's correct and it only happens as part of the condition update endpoint and never as a stand alone action.